catalog/task/update-infra-deployments/0.1/README.md

```diff
@@ -0,0 +1,23 @@
+# update-infra-deployments task
+
+Clones redhat-appstudio/infra-deployments repository, runs a script obtained from the infra-deployment-update-script key in extraData, generates pull-request for redhat-appstudio/infra-deployments repository.
+
+
+## Parameters
+|name|description|default value|required|
+|---|---|---|---|
+|EXTRADATA_JSON_FILE| Path to extraData json file. It should contain a key called 'infra-deployment-update-script' | |true|
+|IMAGES| Path to snapshot json file | | true|
```

Comment:
for consistency can we call this snapshotPath?

Reply:
fixed